travis/minimesosFile

```diff
@@ -55,7 +55,7 @@ minimesos {
         imageName = "containersol/mesos-agent"
         imageTag = "1.0.0-0.1.0"
         loggingLevel = "# INHERIT FROM CLUSTER"
-        portNumber = 5052
+        portNumber = 5051
```

Comment:
Have you confirmed that this works on macos? @pschorf mentioned that he thought it may not